<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctkQtTestingMainWindow</class>
 <widget class="QMainWindow" name="ctkQtTestingMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="ctkSearchBox" name="SearchBox"/>
      </item>
      <item>
       <widget class="ctkAxesWidget" name="AxesWidget" native="true"/>
      </item>
      <item>
       <widget class="ctkVTKRenderView" name="renderView" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="backgroundColor" stdset="0">
         <color>
          <red>51</red>
          <green>51</green>
          <blue>51</blue>
         </color>
        </property>
        <property name="backgroundColor2" stdset="0">
         <color>
          <red>51</red>
          <green>51</green>
          <blue>51</blue>
         </color>
        </property>
        <property name="gradientBackground" stdset="0">
         <bool>false</bool>
        </property>
        <property name="orientationWidgetVisible" stdset="0">
         <bool>true</bool>
        </property>
        <property name="pitchRollYawIncrement" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkRangeSlider" name="RangeSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkRangeWidget" name="RangeWidget" native="true"/>
      </item>
      <item>
       <widget class="ctkSliderWidget" name="SliderWidget" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSpinBox" name="spinBox"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item>
       <widget class="ctkFontButton" name="FontButton"/>
      </item>
      <item>
       <widget class="QPushButton" name="RecordButton">
        <property name="text">
         <string>Record</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioButton">
        <property name="text">
         <string>Pause_playback</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PauseButton">
        <property name="text">
         <string>Pause_record</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PlayBackButton">
        <property name="text">
         <string>PlayBack</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkVTKRenderView</class>
   <extends>QWidget</extends>
   <header>ctkVTKRenderView.h</header>
  </customwidget>
  <customwidget>
   <class>ctkAxesWidget</class>
   <extends>QWidget</extends>
   <header>ctkAxesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkFontButton</class>
   <extends>QPushButton</extends>
   <header>ctkFontButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkRangeSlider</class>
   <extends>QSlider</extends>
   <header>ctkRangeSlider.h</header>
  </customwidget>
  <customwidget>
   <class>ctkRangeWidget</class>
   <extends>QWidget</extends>
   <header>ctkRangeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSearchBox</class>
   <extends>QLineEdit</extends>
   <header>ctkSearchBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
